excise/tests/test_avatax_excise.py

```diff
@@ -9,40 +10,41 @@
 from django.test import override_settings
 from prices import Money, TaxedMoney
 from requests import RequestException
+from dataclasses import asdict
 
-from saleor.shipping.models import ShippingMethodChannelListing
 
-from .....account.models import Address
-from .....checkout import CheckoutLineInfo
-from .....checkout.models import CheckoutLine
-from .....checkout.utils import add_variant_to_checkout, fetch_checkout_lines
-from .....core.prices import quantize_price
-from .....core.taxes import TaxError
-from .....product.models import ProductType, ProductVariant
-from .....warehouse.models import Warehouse
+# from saleor.shipping.models import ShippingMethodChannelListing
```

Comment:
Please clean up the code by removing all commented and unneeded parts of the codes.